app/src/main/java/edu/uc/coffeens/stocktracker/dto/Stock.kt

```diff
@@ -8,6 +8,8 @@ import com.google.gson.annotations.SerializedName
 data class Stock(
     @SerializedName("symbol") var stockTicker: String,
     @SerializedName("company") var stockCompany: String,
+    @SerializedName("Opening Price") var OpeningPrice: String,
+    @SerializedName("Closing Price") var ClosingPrice: String,
```

Comment:
So you guys just have symbol and company which is fine, but someone like me who likes to trade stocks and options I would like the opening and closing prices because it helps me set my strategy for the next day. More of a personal preference but I think it would be a nice feature to add.

Reply:
Is it used anywhere?  If not, it's an incomplete feature.

Are these fields, with this name, available in the data feed?  
- The @SerializedName annotation indicates that it is.
- But... that's unlikely, as the names provided contain spaces.

Verify these fields exist before adding them.